### DISCUSSION TITLE
Dashboard: Add time to first token chart (ttft)

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#3663] [Ideas]
### Describe the feature or potential improvement

Add chart to track ttft over time

**Comment by jansenderr** [comment]
Please!

**Comment by Steffen911** [comment]
@jansenderr This is available via our custom dashboards. You can use the Time to First Token metric on observations:
<img width="1591" alt="image" src="https://github.com/user-attachments/assets/a5e1029d-4bfb-4907-9545-689c8f769c07" />
